### ISSUE TITLE
Instance: The `#chat-reply-info` label is insufficiently descriptive | WCAG 2.2 Violation - There are non-descriptive/insufficiently descriptive labels present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#51]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1. 

### Screenshot

![Screenshot of the \`#chat-reply-info\` element.](https://github.com/user-attachments/assets/328ec850-4292-470c-adf4-107fa839109d)

### Code Reference

```
<form id="chat-commentform" class="chat-form">
	<div class="chat-reply-indicator">
		<div id="chat-reply-info" class="chat-reply-info">
			<div class="chat-reply-indicator-text">
				<svg aria-hidden="true" width="14" height="14" viewBox="0 0 16 16" fill="currentColor">[…]</svg>

				<span>Replying to <strong>User 1</strong></span>
			</div>

			<div class="chat-reply-preview">Another comment </div>
		</div>
		[…]
	</div>
	[…]
</form>
```
